jaeger/charts/linkerd-jaeger/README.md

```diff
@@ -75,6 +75,8 @@ Kubernetes: `>=1.16.0-0`
 | collector.image.pullPolicy | string | `"Always"` |  |
 | collector.image.version | string | `"0.1.11"` |  |
 | collector.jaegerAddr | string | `nil` | address of the jaeger backend to send traces to |
+| createdByAnnotation | string | `"linkerd.io/created-by"` |  |
```

Comment:
These probably got added back in due to an automatic merge? I think they should be removed.